libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -97,6 +97,7 @@ class AP_Motors {
     float               get_roll() const { return _roll_in; }
     float               get_pitch() const { return _pitch_in; }
     float               get_yaw() const { return _yaw_in; }
+    float               get_throttle_out() const { return _throttle_out; }
```

Comment:
I think we should consider removing the default implementation of get_throttle_out() and thus force developers to try to implement it.  Otherwise I just worry that the dynamic notch filter won't work on new frames and we will end up with support calls from users.

Reply:
Setting it to zero will result in a fixed harmonic notch so this will be ok. I will implement this for the other frames as well.